config/core/300-secret.yaml

```diff
@@ -19,6 +19,9 @@ metadata:
   # https://github.com/knative-sandbox/control-protocol/blob/main/pkg/certificates/reconciler/controller.go
   name: serving-certs-ctrl-ca
   namespace: knative-serving
+  labels:
+    serving-certs-ctrl: "data-plane"
+    networking.internal.knative.dev/certificate-uid: "serving-certs"
```

Comment:
+1 for future reference two things need patching due to filtering, this secret since internal encryption relies on control-protocol and test/e2e/domainmapping/domain_mapping_test.go if we enable filtering on net-{kourier, istio}.

Reply:
@skonto just to confirm, domain mapping doesn't need to be updated for this PR?

Reply:
No it does not for now.